src/traveltime_google_comparison/requests/here_handler.py

```diff
@@ -62,6 +62,14 @@ async def send_request(
                         for section in first_route["sections"]
                     )
 
+                    # For some reason, HERE provider returns 0 duration, 0 length
+                    # for some routes in the mountains, but doesn't indicate anywhere
+                    # that it failed. Returning 0 fails `asType(int)` conversion later.
+                    # Example route in UK where this happens:
+                    # "58.61966879999991, -5.0040819999999995","58.578906999999894, -4.880025099999999"
+                    if total_duration == 0:
+                        RequestResult(None)
+
```

Comment:
Unrelated, but fixing an issue with HERE requests. There are 4 weird locations in the UK input file, that are far away in the mountains. Most providers just give pretty random / varied travel_times, but HERE just gives 0, which messes up the analysis code later.